### DISCUSSION TITLE
[feature request]动态中左右切换图片的动态展示 转换成 传统平铺的形式

### DISCUSSION THREAD

**Discussion by oxygenkun** [the1812/Bilibili-Evolved#4645] [功能建议 / Ideas]
只支持左右滑动图片的动态形式：
![image](https://github.com/the1812/Bilibili-Evolved/assets/24502572/09753444-4599-4c28-88c3-2b7ecb6de8da)
> [度过了一个特种兵式的春节！](https://www.bilibili.com/opus/900544549091278849)

图片平铺的形式：
![image](https://github.com/the1812/Bilibili-Evolved/assets/24502572/4dccdcc9-6527-4a2b-9663-0958bea5f9a9)

**Comment by the1812** [comment]
记录一下, 将动态数据中 `modules.module_dynamic.major.opus.style` 置为 `undefined` 即可实现.
